_posts/2018-04-06-all-hands.md

```diff
@@ -0,0 +1,182 @@
+ ---
+layout: post
+title: "The Rust Team All Hands in Berlin: a Recap"
+author: Aaron Turon
+---
+
+Last week we held an "All Hands" event in Berlin, which drew more than 50 people
+involved in 15 different Rust Teams or Working Groups, with a majority being
+volunteer contributors. This was the first such event, and its location reflects
+the current concentration of team members in Europe. The week was a smashing
+success which we plan to repeat on at least an annual basis.
+
+The impetus for this get-together was, in part, our [ambitious plans] to ship
+*Rust, 2018 edition* later this year. A week of facetime was great a great way
+to kick off these efforts!
+
+We've also asked attendees to blog and tweet about their experiences at the
+#RustAllHands hashtag; the Content Team will be gathering up and summarizing
+this content as well.
+
+[ambitious plans]: https://blog.rust-lang.org/2018/03/12/roadmap.html
+
+## Highlights by team
+
+Below we'll go through the biggest items addressed last week. **Note
+that, as always, reaching consensus in a meeting does *not* mean any final
+decision has been made**. All major decisions will go through the usual [RFC
+process].
+
+[RFC process]: https://github.com/rust-lang/rfcs
+
+### Language design
+
+- **Macros**: put together a proposal for the 2018 edition.
+  - Stabilize a forward-compatible subset of procedural macros that explicitly
+    opts out of hygiene (by asking all names to be interpreted at the call
+    site).
+  - Work out a revised API surface for procedural macros based on what we've learned so far.
+  - Stabilize importing macros through normal `use` statements.
+  - **Alex Crichton will write an RFC for the above**.
+- **Extern types**: worked through [remaining issues for stabilization](https://github.com/rust-lang/rust/issues/43467#issuecomment-377521693).
+- **Improvements to `derive`**: a [proposal](https://github.com/rust-lang/rfcs/pull/2385) to make `derive` more ergonomic in Rust 2018.
+- **Best practices**: set up a cross-cutting guidelines effort, encompassing the
+  API guidelines but also including style, lints, and Lang Team recommendations.
+
+### Libraries
+
+- **SIMD**: talked through final steps of stabilization; we hope to [stabilize in 1.27](https://github.com/rust-lang/rust/issues/48556#issuecomment-378184312).
+- **Allocators**: developed a conservative proposal for stabilizing global allocators; **Simon Sapin will post the proposal on the tracking issue**.
+
+### Compiler
+
+- **Tool integration**: extensive discussion and planning about the needs of
+  compiler clients, composable plugins, and the compiler's new query
+  architecture.
+- **Query compilation**: a plan for end-to-end query compilation, i.e. fully-incremental compilation.
+- **libsyntax**: a long-run plan for a revamped libsyntax, shared amongst a variety of tools.
+- **Contribution**: improved contribution experience for the compiler.
+
+### Community
```

Comment:
@skade @Manishearth do you wanna add something here about the conferences session?